appl/smry_appl.cpp

```diff
@@ -2533,6 +2533,25 @@ void SmryAppl::keyPressEvent ( QKeyEvent *event )
             this->update_chart_labels();
         }
     }
+
+    else if (m_smry_loaded && (event->key() == Qt::Key_Home)) {
```

Comment:
Handling of Home and End button also need to be in the [EventFilter function](https://github.com/OPM/qsummary/blob/00df1292197e3052d8060f39e382c99e9fe8a4c0/appl/smry_appl.cpp#L1944).